challenge-programs: add description for UCP season 2 (#175)
pingcap/community: challenge-programs/challenge-program-season-2.md

# Challenge Program Season 2

Welcome to TiDB Challenge Program Season 2.

Season 2 of the TiDB Challenge Program series focuses on the usability of TiDB and its related projects.

Following the principle of “basing directions on users' requirements", we drew most of the requirements of this season from users' votes. These requirements will be presented in the form of tasks in this season. Participants can earn challenge points by claiming tasks. And redeem prizes at the end of the season.

> Season 2 of TiDB Challenge Program officially started on March 02, 2020, and will end on May 30, 2020, lasting 3 months.

Reviewer: ```suggestion
Season 2 of TiDB Challenge Program officially started on March 02, 2020, and will end on June 02, 2020, lasting 3 months.
```

Reviewer: @gingerkidney Please confirm.

> Season 2 of TiDB Challenge Program officially started on March 02, 2020, and will end on June 02, 2020, lasting 3 months.

Author: ```suggestion
Season 2 of TiDB Challenge Program officially started on March 02, 2020, and will end on May 30, 2020, lasting 3 months.
```

We can’t wait to see innovative ideas sparkled. Participants are welcomed to join [TiDB Community Slack Workspace](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE) on the slack. Feel free to contact us in the **#tidb-challenge-program** channel if you have any questions.

## Before participation

1. Create a GitHub account. See [Join GitHub](https://github.com/join) for more information.
2. [Install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) in your local environment.
3. Configure your Git to access Github. See [Set up Git](https://help.github.com/en/github/getting-started-with-github/set-up-git) for more information.

## Sign up

To sign up for the program, [Open an Issue](https://github.com/tidb-challenge-program/register) in the [tidb-challenge-program/register](https://github.com/tidb-challenge-program/register) Repo, in the following format:

> - Title: PCP/Sign Up

Reviewer 2: ```suggestion
- Title: UCP/Sign Up
```

- `difficulty/easy`, `difficulty/medium`, `difficulty/hard`: difficulty level of the issue
- `Component/XX`: product component of the issue

Of course, if you have other ideas of contributions that are not presented under the TODO column, you can raise a proposal by opening a new Issue to the corresponding Repo. Before you do that, make sure you have read the [Contribution Guide](https://github.com/pingcap/community/blob/master/CONTRIBUTING.md) and [Contributor Covenant Code of Conduct](https://github.com/pingcap/community/blob/master/CODE_OF_CONDUCT.md).

 - Raise a proposal to TiDB Repo：[https://github.com/pingcap/tidb/issues/new?labels=type%2Fenhancement&template=feature-request.md](https://github.com/pingcap/tidb/issues/new?labels=type%2Fenhancement&template=feature-request.md)
 - Raise a proposal to TiKV Repo：[https://github.com/tikv/tikv/issues/new?template=feature-request.md](https://github.com/tikv/tikv/issues/new?template=feature-request.md)
 - Raise a proposal to PD Repo：[https://github.com/pingcap/pd/issues/new?labels=type%2Fenhancement&template=feature-request.md](https://github.com/pingcap/pd/issues/new?labels=type%2Fenhancement&template=feature-request.md)

> **Format requirement**: Add the mark `REQ-PCP` at the beginning of the issue title, for example: 

Reviewer 2: ```suggestion
**Format requirement**: Add the mark `REQ-UCP` at the beginning of the issue title, for example: 
```

> `REQ-PCP: Further extract tidb_query into different workspaces`

Reviewer 2: ```suggestion
`REQ-UCP: Further extract tidb_query into different workspaces`
```
